packages/client/src/__tests__/integrations.test.ts

```diff
@@ -0,0 +1,77 @@
+import { httpsCallable } from "@firebase/functions";
+import { createJestSMTPServer } from "jest-smtp";
+
+import { EmailPayload } from "@eisbuk/shared";
+import {
+  DeliveryStatus,
+  ProcessDocument,
+} from "@eisbuk/firestore-process-delivery";
+
+import { CloudFunction } from "@/enums/functions";
+
+import { setUpOrganization } from "@/__testSetup__/node";
+import { functions } from "@/__testSetup__/firestoreSetup";
+
+import { testWithEmulator } from "@/__testUtils__/envUtils";
+import { waitForCondition } from "@/__testUtils__/helpers";
+
+describe("Email sending and delivery", () => {
+  // Dummy data for error testing
+  const to = "saul@gmail.com";
+  const subject = "Subject";
+  const html = "html";
+  const smtpServer = createJestSMTPServer({
+    port: 5000,
+    host: "localhost",
```

Comment:
These should probably be stored in a 'single-source-of-truth' place and imported, rather than repeating a couple of times (throughout files). It's used at least in `setUpOrganization`